packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -98,6 +98,9 @@ export class EpochTracker {
             this.validateEpochFromResponse(epochFromResponse, fetchType);
             return response;
         } catch (error) {
+            if (epochFromResponse === undefined) {
```

Comment:
Is this code here to get it from response when response failed? I.e. we have not had a chance to get to line 97?
Adding some comment on why this code is here might be useful for future

Reply:
This is to actually get from error. because we always add serverEpoch in the response